### PR TITLE
352 - Fast Track Bot

### PR DESCRIPTION
**Pull request by jimmyhmiller**

### CODE REVIEW

**Comment by jimmyhmiller** [comment]
yay

**Comment by ArchmageInc** [comment]
yay

**Comment by ArchmageInc** [comment]
Apparently, we have it set to >= 50% for quorum according to the Bot

**Comment by jimmyhmiller** [comment]
> 1. A proposal must be voted on by more than 50% of active players (a quorum) in order to be resolved (accepted or rejected) unless otherwise noted in the rules. Any time after a proposal has reached a quorum it may be resolved by its creator. A rule-change is a proposal.

**Comment by ArchmageInc** [comment]
I am not saying the bot is correct. I am only saying that the bot assigned the "Has Quorum" label with 2/4 votes.

**Comment by dilbertKocik** [comment]
Wait I'm confused. Why would we want different rules for adding features to the bot if adding features to the bot is considered a rule-change? If that isn't the case, then doesn't this rule have to take precedence over 350 as well? Otherwise it won't actually be able to take affect.

**Comment by jimmyhmiller** [comment]
@dilbertKocik Okay. So bot features aren't rule-changes, they just use the same voting rules as them (according to 350).

That means that 311 applies to bot features. 351 conflicts with 311, 311 is earlier so it by default has precedence, but by stating that 351 takes precedence over 311, the conflict is resolved.

351 and 350 don't conflict. All the rules that apply for rule changes do apply for bot feature adding. That doesn't mean there can't also be rules that only apply to bot features. Applying a rule involves resolving precedence, just because there is a law that takes precedence over another law doesn't mean we aren't applying the rule, it just means its proper application is to be surpassed by the law that takes precedence.

TL;DR - I want to get bot changes passed. This allows it. Please do it, I already have staged changes.

**Comment by ArchmageInc** [comment]
Clear as mud?

**Comment by dilbertKocik** [comment]
Well I'll give you quorum so you can pass it but I disagree with the reason for the rule so I'm voting nay. You introduced this problem yourself by making bot changes require voting. If that's a problem then it makes more sense to simply remove that rule. You wouldn't even be introducing this rule if everyone was responding to your PRs. I just see it as something that is addressing a symptom rather than the actual problem. Kind of like how @ArchmageInc would call everyone out as being inactive.

**Comment by dilbertKocik** [comment]
nay

**Comment by jimmyhmiller** [comment]
@dilbertKocik I agree.
